reflection/testdata/dynamic.pb.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package testdata
+
+var File_dynamic_proto_rawDesc = []byte{
```

Comment:
Document how this file/this variable is generated.
And golint might complain about this variable name (because of the underscores)

You will also need to handle it specially otherwise the script will try to regenerate.
https://github.com/grpc/grpc-go/blob/master/regenerate.sh#L32
https://github.com/grpc/grpc-go/blob/master/regenerate.sh#L120

Or, rename folder `grpc_testingv3` to `grpc_testing_not_regenerate`, and move dynamic.proto there.
So that folder contains the two special proto files that don't need to regenerated.